packages/p2p/src/components/app.jsx

```diff
@@ -70,392 +50,157 @@ class App extends React.Component {
         };
 
         waitWS('authorize').then(() => {
-            this.ws_subscriptions = {
+            general_store.ws_subscriptions = {
```

Comment:
Could we move this logic to the store as well? e.g. we create a `onMount` function in the store which will subscribe to `p2p_advertiser_info` + `p2p_order_list`.

Reply:
Done :)